docs/sphinx/manual/building_hpx.rst

```diff
@@ -1269,3 +1271,25 @@ How to install |hpx| on Fedora distributions
 .. include:: ../../generated/cmake_toolchains.rst
 
 .. include:: ../../generated/cmake_variables.rst
+
+.. _arch_installation
+
+How to install |hpx| on Arch distributions
+------------------------------------------
+
+* Install all packages for minimal installation:
+
+  .. code-block:: bash
+
+     sudo pacman -S gcc clang cmake boost hwloc gperftools
+
```

Comment:
Could you write documentation instead of docs, please?

Reply:
I meant literally changing the word "docs" to "documentation" (just below these comments). I just prefer writing out full words unless there's a good reason not to, but you can leave it as it is as well if you'd like.